JeongJaeHeon/db/migrations/20221126023026_create_users_tables.sql

```diff
@@ -0,0 +1,13 @@
+-- migrate:up
+CREATE TABLE users (
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    name VARCHAR(100) NOT NULL,
+    email VARCHAR(200) NOT NULL,
+    password VARCHAR(1000) NOT NULL,
+    profile_image VARCHAR(1000) NULL,
+    created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP
+)
```

Comment:
회원 정보 중 아이디 혹은 이메일은 중복 불가능하게 설정된 경우가 많습니다!

현재 설정된 컬럼에서는 `email`을 중복 불가능하게 설정하여 유저의 정보가 중복 저장되는 것을 방지할 수 있습니다.

아래 sql문을 참고하여 `email` 컬럼에 `unique` 속성을 추가해주세요!

제약 조건을 통해 제약 조건의 이름과 컬럼을 설정헐 수 있습니다. 삭제, 비활성화 등의 관리를 위해서 제약조건에 이름을 지정해주는 것이 좋습니다. 

```suggestion
CREATE TABLE users (
    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
    name VARCHAR(100) NOT NULL,
    email VARCHAR(200) NOT NULL,
    password VARCHAR(1000) NOT NULL,
    profile_image VARCHAR(1000) NULL,
    created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
    updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
    CONSTRAINT users_email_ukey UNIQUE (email);
)
```